src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/mapView.js

```diff
@@ -74,11 +75,12 @@ class MapViewVisualization extends Visualization {
         while (this.dom.firstChild) {
             this.dom.removeChild(this.dom.lastChild);
         }
-
+        MAP_ID += 1;
+        const id      = "map_" + MAP_ID;
         const width   = this.dom.getAttributeNS(null,"width");
         const height  = this.dom.getAttributeNS(null,"height");
         const mapElem = document.createElement("div");
-        mapElem.setAttributeNS(null,"id"   ,"map");
+        mapElem.setAttributeNS(null,"id"   ,id);
```

Comment:
Are you sure you want to change the ID on EVERY time the map gets data? This is inside of function `onDataReceived`. I feel its not the right thing to do

Reply:
fixed